w06h01/test/pgdp/LoopTest.java

```diff
@@ -22,7 +24,12 @@ public void testUsageOfLoops() {
                 "src/pgdp/datastructures/lists/RecIntListElement.java"
         };
 
-        String[] notAllowedKeywords = new String[] { "for", "while", "Stream" };
+        String[] notAllowedRegExp = new String[] {
+                "for\s*[(]{1}",
+                "while\s*[(]{1}",
+                "Stream.of",
+                "Stream<",
```

Comment:
There are still thousands ways to use Streams. However, I think for the beginning is this enough

Reply:
I think "Arrays.stream(" would be a good candidate too as we already got to know the Arrays class in the past

Reply:
Added ✅